### DISCUSSION TITLE
How do I set endpoint exception alarm

### DISCUSSION THREAD

**Discussion by liuqiyang7** [apache/skywalking#10697] [Q&A] [answered]
This alarm is triggered when a node in the request is abnormal

**Comment by wu-sheng** [comment, marked as answer]
Are you asking for an endpoint or something? What is a node?

**Reply by liuqiyang7** [reply]
The endpoint or service triggers an alarm as soon as it fails

**Reply by wu-sheng** [reply]
I don't know what do you mean fail. Please don't create a concept that you are the only one understanding.

Alerting is metric-oriented, that is the only thing I can say for now.

**Reply by liuqiyang7** [reply]
![image](https://user-images.githubusercontent.com/63906369/232704365-b9d53a70-8bf4-4273-8828-3940a1f55eac.png)

**Reply by wu-sheng** [reply]
No, one single request would NOT trigger an alert. That is totally wrong from operation and SRE perspective

**Reply by liuqiyang7** [reply]
ok,thanks
